dialog.php

```diff
@@ -19,6 +19,9 @@
             case 'diff':
                 include('templates/diff.html');
                 break;
+            case 'fetch':
+                include('templates/fetch.html');
```

Comment:
This works, but why create a new dialog? Better implement the logic into the [remotes dialog](https://github.com/Andr3as/Codiad-CodeGit/blob/master/templates/remote.html).